ticket_booking_app/run_script

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+echo "Ticket_booking_app"
+
+echo "mysql root password"
+
+mysql -u root -p -e "create database cinema_db; CREATE USER 'cinema_user'@'localhost' IDENTIFIED BY '1234'; GRANT ALL PRIVILEGES ON cinema_db.* TO cinema_user@localhost; ALTER DATABASE cinema_db CHARACTER SET utf8mb4 COLLATE utf8mb4_polish_ci; "
```

Comment:
Rozwiązanie zakłada, że masz zainstalowanego mysql'a.  Nie lepiej zrobić to na jakiejś in-memory DB?